### DISCUSSION TITLE
Coups illégaux

### DISCUSSION THREAD

**Discussion by wilchel** [pascalaubry/papi-web#33] [Ideas]
Il parait risquer que des joueurs par erreur ajout des coups illégaux
Peut etre prévoir un affichage des tables dédiées aux arbitres pour ajouter les coups illégaux et laisser sur la page des joueurs l information de coups illégaux

**Comment by Amaras** [comment]
Je suis d'accord que ça peut être risqué.
Si seuls les arbitres peuvent rentrer les résultats, les joueurs ne peuvent pas rentrer des coups illégaux par erreur, et cet affichage est tout indiqué.
La gestion des autorisations est quelque chose qui est déjà en réflexion depuis un moment, voir [ce fichier](https://github.com/papi-web-org/papi-web/blob/dev/docs/95-auth.md)
